examples/prism-system_verilog.html

```diff
@@ -0,0 +1,104 @@
+<h1>System Verilog</h1>
+<p>To use this language, use the class "language-system_verilog".</p>
```

Comment:
Here, you might want to add a paragraph explaining that the component is also compatible with Verilog.